languages/csharp/exercises/concept/overflow/.docs/introduction.md

```diff
@@ -0,0 +1,21 @@
+Arithmetic overflow occurs when a computation such as an arithmetic operation or type conversion results in a value that is greater than the capacity of the receiving type.
+
+Expressions of type `int` and `long` and their unsigned counterparts will quietly wrap around under these circumstances.
+
+The behavior of integer computations can be modified by using the `checked` keyword. When an overflow occurs within a `checked` block an instance of `OverflowException` is thrown.
+
+```csharp
+var xx = new Random().Next();
+checked
+{
+    int expr = int.MaxValue * xx;   // OverflowException is thrown
+}
+
+// or
+
+int expr2 = checked(int.MaxValue * xx);     // OverflowException is thrown
```

Comment:
We could get rid of the `Random` bit by introducing a variable, with which we can increment by one to hopefully even more clearly show the overflow:

```suggestion
int one = 1;
checked
{
    int expr = int.MaxValue + one;   // OverflowException is thrown
}

// or

int expr2 = checked(int.MaxValue + one);     // OverflowException is thrown
```